src/components/Button.tsx

```diff
@@ -33,7 +33,7 @@ export class Button extends Component<ButtonProps, ButtonState> {
     public static defaultProps: Partial<ButtonProps> = {
         text: 'Button item',
         icon: undefined,
-        fontSize: 14,
+        fontSize: 1,
```

Comment:
Correct me if I am wrong. I think 1 rem = 16px so if the previous fontSize was 14, I think we should use 0.875 here.

Reply:
Done!